libopenage/renderer/opengl/buffer.cpp

```diff
@@ -21,6 +21,18 @@ GlBuffer::GlBuffer(size_t size, GLenum usage)
 	glBufferData(GL_ARRAY_BUFFER, size, 0, usage);
 }
 
+GlBuffer::GlBuffer(const float *data, size_t size, GLenum usage)
```

Comment:
This constructor seems unnecessary, since we can just cast `float*` to `uint8_t*`. The meaning of `uint8_t` is not necessarily that it has to be usable as unsigned ints, but rather just that it's some number of bytes.

Reply:
Sure. That constructor again is not being used anyway. Lots of testing code left off. I'll remove that as wwel